lib/lotus/repository.rb

```diff
@@ -283,7 +283,8 @@ def persist(entity)
       #
       #   ArticleRepository.create(article) # no-op
       def create(entity)
-        unless entity.id
+        unless entity.persisted?
+          update_timestamps(entity)
```

Comment:
the Lotus conventional naming for private is appending `_` prefix, so please rename this method to `_update_timestamps(entity)`